### ISSUE TITLE
Tag names CAN contain the / character

### ISSUE THREAD

**Issue by 0ion9** [oniony/TMSU#180]
(contrary to what help and the wiki say)

Just found this due to an automation bug.
Test case:

```
cd /tmp/
tmsu init .
touch foo
tmsu tag foo foo/bar/baz
tmsu tags
tmsu files foo/bar/baz
```

In all respects these behave as if / is acceptable -- the tagging goes through, and the 'tmsu files' command returns the correct filename.

I made similar checks for backslash; it is possible to apply a tag containing backslash, but it cannot be queried using `files`. 
Neither tags containing / nor \\ can be chdired to in the VFS.

Overall conclusion:
* validation may not be firing at all on 'tmsu tag' (and untag? I didn't test that)
* validation is incomplete for `tmsu files`

TMSU version: "0.8.0", latest git (rebuilt today)

**Comment by oniony** [comment]
Yes, this change is in the release notes for v0.7.0. Slashes are allowed
but use is discouraged as the VFS does not support them. A similar looking
Unicode character is substituted on the VFS. You should be able to change
to the directories if you tab complete the name.

The wiki and help are likely out of date. I'll leave this open to fix that.

**Comment by 0ion9** [comment]
I see. However, judging from your reply I feel as if I may need to point out that **backslash** is also (buggily/inconsistently) allowed, and unless I'm being confused by escaping rules, that seems like it is definitely a wrong behaviour; it should either be always accepted or always rejected.

**Comment by oniony** [comment]
It should, but it's a fringe case and I'm struggling for time to fix
anything at the moment.

**Comment by oniony** [comment]
Backslash seems to work fine to me. Or do you mean in the VFS?

    tmsu tag somefile \\\\
    tmsu files \\\\
